src/progressive-profiling/ProgressiveProfiling.jsx

```diff
@@ -45,7 +47,7 @@ const ProgressiveProfiling = (props) => {
   const [values, setValues] = useState({});
   const [openDialog, setOpenDialog] = useState(false);
   const [showRecommendationsPage, setShowRecommendationsPage] = useState(false);
-
+  const authenticatedUser = getAuthenticatedUser();
```

Comment:
Should we memoize this value?

Reply:
I don't think we should memoize it since the get authenticated user JWT tokens refresh token for authentication and we do want to monitor live authentication for the cases where the user gets unauthenticated for some reason. right?
memoizing it will return the fetched value unless the whole component is not re-mounted or we provide a specific dependency. Also, the purpose of memoizing is the memoize the complex time taking calculations, not the API calls. what do you say?